### DISCUSSION TITLE
What is the "remember" objects in the sample?

### DISCUSSION THREAD

**Discussion by csbenz** [patrykandpatrick/vico#528] [Questions] [answered]
For example
https://github.com/patrykandpatrick/vico/blob/master/sample/src/main/java/com/patrykandpatrick/vico/sample/showcase/charts/Chart8.kt

**Comment by patrickmichalik** [comment, marked as answer]
Hello! `rememberCartesianChart`, `rememberColumnCartesianLayer`, and so on are factory functions that use Jetpack Compose’s `remember`. `rememberCartesianChart`, for example, creates and remembers a `CartesianChart` instance. This is in line with [Google’s naming guidelines](https://android.googlesource.com/platform/frameworks/support/+/androidx-main/compose/docs/compose-api-guidelines.md#naming-composable-functions-that-remember-the-objects-they-return). You can find more information on Vico’s APIs in [the API reference](https://patrykandpatrick.com/vico/api/2.0.0-alpha.5/).
